variables.tf

```diff
@@ -7,16 +7,16 @@ variable "ecr_name" {
   default = "isengardecr"
 }
 
-variable "dynamodb_table_name"{
-  default = "pets-table"
+variable "dev-project-terraform-state-lock-dynamo"{
+  default = "pets-table-1"
```

Comment:
default values doesn't have to be changed, any variable definition have to be updated in dev.tfvars or whatever environment have to be updated

Reply:
reverted